### DISCUSSION TITLE
Bind magit-notes to 'N' in magit-status-mode-map

### DISCUSSION THREAD

**Discussion by ghost** [magit/magit#4796] [Support] [answered]
Currently magit-notes is bound to 'T'.  The more mnemonic 'N' is unbound.

**Comment by tarsius** [comment, marked as answer]
While it might make less sense, `T` was chosen because one can think of notes as a concept similar to tags, which is bound to `t`.

I haven't used `N` to keep it available for some sort of "next"-command, but because there just aren't enough keys available, I ended up using it for something else: `forge-dispatch`. That isn't bound in `magit-mode-map` unless you install `forge`, but there is a note in the definition of `magit-mode-map` about it being reserved for that command.

**Comment by ghost** [comment]
Thanks Jonas.  The relationship to the concept of tags was the missing
clue.  Now I have my mnemonic :-)
/john

On Fri, Oct 21, 2022 at 9:15 AM Jonas Bernoulli
***@***.***> wrote:
>
> While it might make less sense, T was chosen because one can think of notes as a concept similar to tags, which is bound to t.
>
> I haven't used N to keep it available for some sort of "next"-command, but because there just aren't enough keys available, I ended up using it for something else: forge-dispatch. That isn't bound in magit-mode-map unless you install forge, but there is a note in the definition of magit-mode-map about it being reserved for that command.
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.Message ID: ***@***.***>



-- 
John Yates
505 Tremont St, #803
Boston, MA 02116
